docs/monthday.md

```diff
@@ -13,20 +13,22 @@ A `Temporal.MonthDay` can be converted into a `Temporal.Date` by combining it wi
 
 ## Constructor
 
-### **new Temporal.MonthDay**(_isoMonth_: number, _isoDay_: number) : Temporal.MonthDay
+### **new Temporal.MonthDay**(_isoMonth_: number, _isoDay_: number, _refISOYear_: number = 1972) : Temporal.MonthDay
 
 **Parameters:**
 - `isoMonth` (number): A month, ranging between 1 and 12 inclusive.
 - `isoDay` (number): A day of the month, ranging between 1 and 31 inclusive.
+- `refISOYear` (optional number): A reference year, used for disambiguation when implementing other calendar systems.
+  This parameter can usually be omitted.
```

Comment:
(here and below)

"This parameter can usually be omitted" is wishy-washy.  Give the condition when you can omit it:

"Omit this option unless using a non-ISO-8601 calendar."